backend/src/main/java/com/ody/meeting/controller/MeetingController.java

```diff
@@ -30,39 +33,22 @@
 public class MeetingController implements MeetingControllerSwagger {
 
     private final NotificationService notificationService;
+    private final MeetingService meetingService;
+    private final MateService mateService;
 
     @Override
     @GetMapping("/meetings/me")
     public ResponseEntity<MeetingSaveResponses> findMine(@AuthMember Member member) {
-        return ResponseEntity.ok(new MeetingSaveResponses(
-                        List.of(
-                                new MeetingSaveResponse(
-                                        1L,
-                                        "우테코 16조",
-                                        LocalDate.parse("2024-07-15"),
-                                        LocalTime.parse("14:00"),
-                                        "서울 송파구 올림픽로35다길 42",
-                                        "37.515298",
-                                        "127.103113",
-                                        2,
-                                        List.of(new MateResponse("조조"), new MateResponse("제리")),
-                                        "초대코드"
-                                ),
-                                new MeetingSaveResponse(
-                                        2L,
-                                        "우테코 15조",
-                                        LocalDate.parse("2024-07-17"),
-                                        LocalTime.parse("14:00"),
-                                        "서울 송파구 올림픽로35다길 42",
-                                        "37.515298",
-                                        "127.103113",
-                                        2,
-                                        List.of(new MateResponse("카키"), new MateResponse("제리")),
-                                        "초대코드"
-                                )
-                        )
-                )
-        );
+
+        List<Meeting> memberMeetings = meetingService.findAllMeetingsByMember(member);
+        List<MeetingSaveResponse> saveResponses = memberMeetings.stream()
+                .map(this::makeSaveResponse)
+                .toList();
```

Comment:
[제안]
체이닝으로 한 번에 작성하는건 어떨까요?
```suggestion
        List<MeetingSaveResponse> saveResponses = meetingService.findAllMeetingsByMember(member)
                .stream()
                .map(this::makeSaveResponse)
                .toList();
```